src/Utopia/Messaging/Adapters/SMS/46elks.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace Utopia\Messaging\Adapters\SMS;
+
+use Utopia\Messaging\Adapters\SMS as SMSAdapter;
+use Utopia\Messaging\Messages\SMS;
+
+// Reference Material
+// https://46elks.com/docs/send-sms
+class fortysixelks extends SMSAdapter
```

Comment:
Should be Pascal Case
```suggestion
class FortySixElks extends SMSAdapter
```